Directory.Build.props

```diff
@@ -6,9 +6,18 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <CppWinRTVerbosity>high</CppWinRTVerbosity>
+    <CsWinRTVerbosity>high</CsWinRTVerbosity>
     <MicrosoftWinUIVersion>3.0.0-preview1.200515.3</MicrosoftWinUIVersion>
     <!-- Temporary fallback until Microsoft.WinUI nuget supports Net5 TFM -->
     <AssetTargetFallback>netcoreapp5.0</AssetTargetFallback>
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <LangVersion>preview</LangVersion>
+    <NoWarn>8305;0618</NoWarn>
+    <GenerateTestProjection Condition="'$(GenerateTestProjection)$(Configuration)' == 'Release'">true</GenerateTestProjection>
+    <GenerateTestProjection Condition="'$(GenerateTestProjection)' == ''">false</GenerateTestProjection>
+    <CsWinRTGenerateProjection>$(GenerateTestProjection)</CsWinRTGenerateProjection>
+    <!--<CsWinRTGeneratedFilesDir>$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files', '$(TargetFramework)'))</CsWinRTGeneratedFilesDir>-->
```

Comment:
Do we want to leave this comment in case of _something_?

Reply:
it's a TODO for me - to return the cswinrt repo to easily located generated files